tests/e2e/gc_block_resize_retain_policy.go

```diff
@@ -725,6 +741,317 @@ var _ = ginkgo.Describe("[csi-guest] Volume Expansion Tests with reclaimation po
+		svcPVC, err := svcClient.CoreV1().PersistentVolumeClaims(svNamespace).Get(ctx, svpvcName, metav1.GetOptions{})
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		svcPV := getPvFromClaim(svcClient, svNamespace, svpvcName)
+		verifyBidirectionalReferenceOfPVandPVC(ctx, svcClient, svcPVC, svcPV, fcdID)
+
+		pvcNew, pvNew := createStaticPVandPVCinGuestCluster(client, ctx, namespace, svpvcName, diskSize, storageclass, v1.PersistentVolumeReclaimRetain)
+		isGC1pvcCreated := true
+		isGC1pvCreated := true
+
+		defer func() {
+			if isGC1pvcCreated {
+				ginkgo.By("Deleting the gc PVC")
+				err = fpv.DeletePersistentVolumeClaim(client, pvcNew.Name, namespaceNewGC)
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			}
+
+			if isGC1pvCreated {
+				ginkgo.By("Deleting the gc PV")
+				err = fpv.DeletePersistentVolume(client, pvNew.Name)
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+				err = fpv.WaitForPersistentVolumeDeleted(client, pvNew.Name, poll, pollTimeoutShort)
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			}
+
+		}()
+
+		ginkgo.By("Trigger offline expansion")
+		currentPvcSize := pvcNew.Spec.Resources.Requests[v1.ResourceStorage]
+		newSize := currentPvcSize.DeepCopy()
+		newSize.Add(resource.MustParse("1Gi"))
+		framework.Logf("currentPvcSize %v, newSize %v", currentPvcSize, newSize)
+		pvcNew, err = expandPVCSize(pvcNew, newSize, client)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(pvcNew).NotTo(gomega.BeNil())
+
+		ginkgo.By("Waiting for controller volume resize to finish")
+		err = waitForPvResizeForGivenPvc(pvcNew, client, totalResizeWaitPeriod)
+		framework.ExpectNoError(err, "While waiting for pvc resize to finish")
+
+		ginkgo.By("Checking for conditions on pvc")
+		pvcNew, err = waitForPVCToReachFileSystemResizePendingCondition(client, namespace, pvcNew.Name, pollTimeout)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Deleting the gc PVC")
+		err = fpv.DeletePersistentVolumeClaim(client, pvcNew.Name, namespace)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		isGC1pvcCreated = false
+
+		ginkgo.By("Deleting the gc PV")
+		err = fpv.DeletePersistentVolume(client, pvNew.Name)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		err = fpv.WaitForPersistentVolumeDeleted(client, pvNew.Name, poll, pollTimeoutShort)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		isGC1pvCreated = false
+
+		ginkgo.By("Check SVC PVC exists")
+		svcPVC = getPVCFromSupervisorCluster(svcPVC.Name)
+		newsizeAfterTriggeringOfflineExpansion := svcPVC.Spec.Resources.Requests[v1.ResourceStorage]
+		newsize := sizeInMb(newsizeAfterTriggeringOfflineExpansion)
+		size := strconv.FormatInt(newsize, 10)
+		framework.Logf("newsizeAfterTriggeringOfflineExpansion size: %s", size)
+
+		pvcNew, pvNew, pod, _ := createStaticPVandPVCandPODinGuestCluster(client, ctx, namespace, svcPVC.Name, "3Gi", storageclass, v1.PersistentVolumeReclaimDelete)
```

Comment:
Run the below command to identify the lines which are exceeding 120 chars 
golangci-lint run --enable=lll 

wrap the line to 120 characters

Reply:
Updated on all the new tests
kramachandr-a01:vsphere-csi-driver kramachandra$ golangci-lint run --enable=lll
kramachandr-a01:vsphere-csi-driver kramachandra$

Reply:
updated on all the new tests 

kramachandr-a01:vsphere-csi-driver kramachandra$ golangci-lint run --enable=lll
kramachandr-a01:vsphere-csi-driver kramachandra$